src/Datasource/RepositoryInterface.php

```diff
@@ -24,6 +24,7 @@
     /**
      * Returns the table alias or sets a new one
      *
+     * @deprecated 3.4.0 Use setAlias()/getAlias() instead.
```

Comment:
We should add a doc string annotation for the new methods. And implement those methods where possible.